### PR TITLE
Hods 425 : Updating to hmrc-mongo

### PR DESCRIPTION
**Pull request by parkeradam**

### CODE REVIEW

File: app/uk/gov/hmrc/individualsemploymentsapi/cache/v2/ShortLivedCache.scala [hunk 1]
```diff
@@ -16,85 +16,44 @@
...
-      "cacheV2.key"
-    )
-    .getOrElse("individuals-employments")
-
-}
+  override val cacheConfig: CacheRepositoryConfiguration, configuration: Configuration, mongo: MongoComponent)
+                               (implicit ec: ExecutionContext)
+  extends BaseCache(cacheConfig, configuration, mongo)
+
+//@Singleton
```
Comment (ajchilts) [inline app/uk/gov/hmrc/individualsemploymentsapi/cache/v2/ShortLivedCache.scala:32]: Can we remove commented out code?

File: app/uk/gov/hmrc/individualsemploymentsapi/cache/v2/CacheRepositoryConfiguration.scala [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ */
+
+package uk.gov.hmrc.individualsemploymentsapi.cache.v2
+
+import play.api.Configuration
+import uk.gov.hmrc.individualsemploymentsapi.cache.{CacheRepositoryConfiguration => BaseConig}
+
+import javax.inject.Inject
+
+class CacheRepositoryConfiguration @Inject()(configuration: Configuration) extends BaseConig {
```
Comment (ajchilts) [inline app/uk/gov/hmrc/individualsemploymentsapi/cache/v2/CacheRepositoryConfiguration.scala:24]: Can I just check with you that this is for V2 only? And that V1 still caches in its original mongo instance and repo name?

**Review by ajchilts — CHANGES_REQUESTED** [review]
@parkeradam  - very good. Just a few small comments to address.

File: app/uk/gov/hmrc/individualsemploymentsapi/cache/v2/CacheRepositoryConfiguration.scala (hunk 2 above)
Comment (parkeradam) [reply app/uk/gov/hmrc/individualsemploymentsapi/cache/v2/CacheRepositoryConfiguration.scala:24]: @ajchilts From what i've been able to tell its in the same table and section as the original. I've ran regression before (master) and after (this branch) and from what I can see it creates the same collections and columns. The locations are decided from config which is done by the individual config sections.

**Review by ajchilts — APPROVED** [review]

**Pull request merged**